src/gui/qgsrasterlayersaveasdialog.h

```diff
@@ -71,6 +71,7 @@ class GUI_EXPORT QgsRasterLayerSaveAsDialog: public QDialog, private Ui::QgsRast
     bool tileMode() const;
     bool addToCanvas() const;
     QString outputFileName() const;
+    QString outputLayerName() const;
```

Comment:
Needs docs, \since tag

Reply:
Already done. Thanks for the super fast review! It's midnight here, I'll address it tomorrow :)